spec/javascripts/components/story/expanded_story/expanded_story_controls_spec.js

```diff
@@ -55,76 +55,125 @@ describe('<ExpandedStoryControls />', () => {
     });
   });
 
-  describe('when the user click on cancell', () => {
+  describe('when the user clicks on cancel', () => {
+    beforeEach(function() {
+      sinon.stub(window, 'confirm')
+    });
+
+    afterEach(function() {
+      window.confirm.restore();
+    });
+
     it('triggers the toggle callback', () => {
-      const onCancel = sinon.spy();
+      const handleCancel = sinon.spy();
 
       const wrapper = shallow(
         <ExpandedStoryControls
           readOnly={false}
-          onCancel={onCancel}
+          onCancel={handleCancel}
         />
       );
 
       wrapper.find('.cancel').simulate('click');
 
-      expect(onCancel).toHaveBeenCalled();
+      expect(handleCancel).toHaveBeenCalled();
```

Comment:
We should add an assertion that checks `window.confirm` is not being called as well, what do you think?